x-pack/filebeat/module/cisco/asa/test/asa-fix.log-expected.json

```diff
@@ -306,7 +306,7 @@
         "event.kind": "event",
         "event.module": "cisco",
         "event.original": "%ASA-4-313009: Denied invalid ICMP code 9, for Inside:10.255.0.206/8795 (10.255.0.206/8795) to identity:10.12.31.51/0 (10.12.31.51/0), ICMP id 295, ICMP type 8",
-        "event.outcome": "deny",
+        "event.outcome": "failure",
```

Comment:
event.outcome in this case should reference the event form the observers perspective, so when the connection is successfully blocked the event.outcome should be "success" (it doesn't really make sense from a FW perspective, but if you had something non-inline it makes more sense)